concordium-sdk/src/test/java/com/concordium/sdk/transactions/TransactionFactoryTest.java

```diff
@@ -43,4 +43,16 @@ public  void testCanDeployModuleFactory() {
         val builder = TransactionFactory.newDeployModule();
         assertNotNull(builder);
     }
+
+    @Test
+    public  void testCanTransferScheduleFactory() {
+        val builder = TransactionFactory.newScheduleData();
```

Comment:
Why is this called `newScheduleData´ ? What does the ´data´mean?  I suggest naming them ´newScheduledTransfer` and ´newScheduledTransferWithMemo`.

Reply:
Changed the function names.